frontend/apps/app/app/(app)/app/organizations/[organizationId]/settings/general/GeneralPage.tsx

```diff
@@ -0,0 +1,203 @@
+'use client'
```

Comment:
For security reasons, we do not want to call supabase in the browser ('use client'). Can you switch to Server Component and Server Actions after this? You can merge them now.